app/services/set_school_local_authority.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SetSchoolLocalAuthority < BaseService
+  def call
+    if school.local_authority&.code != la_code
+      ActiveRecord::Base.transaction do
+        school.latest_school_authority&.update!(end_year: start_year)
+        SchoolLocalAuthority.create!(school: school,
+                                     local_authority: LocalAuthority.find_by(code: la_code),
+                                     start_year: start_year)
```

Comment:
I wonder if a year is granular enough, if we're updating this daily instead of yearly. Would be good to check in with the LPDOB folks to see what they need here

Reply:
LPDOB?
This is only likely to happen where we've had revisions to LAs like splitting Northamptonshire, there are a few awaiting to be processed I think but unlikely to be a common occurrence.

Reply:
Lead Partner team.

Reply:
yes, the catchily named Lead Providers, DfE, and Other Bodies team

Reply:
I asked the question but have had no reply so I guess yearly is still OK.